diff --git a/serializable/__init__.py b/serializable/__init__.py
index 6eef707..cc91f8a 100644
--- a/serializable/__init__.py
+++ b/serializable/__init__.py
@@ -355,7 +355,7 @@ def from_json(cls: Type[_T], data: Dict[str, Any]) -> Optional[_T]:
                     f'There was an AttributeError deserializing JSON to {cls} the Property {prop_info}: {e}'
                 ) from e
 
-        _logger.debug('Creating $s from %s', cls, _data)
+        _logger.debug('Creating %s from %s', cls, _data)
 
         return cls(**_data)
 
